src/containers/Review/notes/NewCommentForm.tsx

```diff
@@ -31,7 +31,7 @@ const NewCommentForm: React.FC<{
       setError(strings.REVIEW_NOTES_COMMENT_ERROR)
       return
     }
```

Comment:
This shouldn't be in here. It's reverting changes that I've already done and merged. Weird.

Anyway, can you just copy and paste this whole file from develop into here to make it correct again?

Or maybe you can merge from develop and that might fix it?

Reply:
Oh, it's due to my rebase.... I was trying to make this up-to-date with latest changes from other PRs...